sdk/sdk-api/src/main/java/mil/emp3/api/utils/ColorUtils.java

```diff
@@ -41,4 +42,26 @@ public static String colorToString(IGeoColor oColor) {
         
         return sStr;
     }
+
+    /**
+     * Returns the IGeoColor of the given hex string
+     * @param hexColorValue - Hex color in String form. Assuming the form of #RRGGBB
+     * @return - IGeoColor object representing the given string.
+     */
+    public static IGeoColor stringToColor(final String hexColorValue) {
+        final int radix = 16;
+        try {
+            int red = Integer.valueOf(hexColorValue.substring(1, 3), radix);
```

Comment:
If the string is empty or less than 3 characters this method will throw an index out of bounds exception.  I guess that is why you have a catch general exception.  It would be better to throw an Illegal Argument exception by checking the length of the string prior to parsing it to integers

Reply:
Fixed